app/views/questions/_answer.html.erb

```diff
@@ -59,7 +59,12 @@
         "><span class="caret"></span>
         </div>
     <% end %>
-       <a href="<% if current_user %>'/answer_like/likes/<%= answer.id %>'<% else %>'#'<% end %>" data-remote ="true" tooltip-title="Helpful? Like it" class="btn btn-default btn-sm btn-like" id="answer-like-button-<%= answer.id %>" <% unless current_user %> popover-content="You must be logged in to Like the answer" <% end %> ><span id="answer-like-star-<%= answer.id %>" class="fa fa-star<% if current_user && !answer.liked_by(current_user.uid) %>-o<% end %>"></span> <span id="answer-like-count-<%= answer.id %>"><%= answer.likers.length %></span> Likes</a>
+       <a href="<% if current_user %>'/answer_like/likes/
+          <%= answer.id %>'<% else %>'#'<% end %>" data-remote ="true" tooltip-title="Helpful? Like it" class="btn btn-default btn-sm btn-like" id="answer-like-button-<%= answer.id %>"
+       <% unless current_user %> popover-content="You must be logged in to Like the answer" 
+          <% end %> ><span id="answer-like-star-<%= answer.id %>" class="fa fa-star<% if current_user && !answer.liked_by(current_user.uid) %>-o<% end %>">
```

Comment:
Here also, please shift end and closing bracket `>` at line 64
So, this line would be
```suggestion
          <span id="answer-like-star-<%= answer.id %>" class="fa fa-star<% if current_user && !answer.liked_by(current_user.uid) %>-o<% end %>">
```